src/components/header/header.scss

```diff
@@ -44,13 +50,34 @@
   }
 }
 
-@media (max-width: 920px) {
+@media (max-width: 640px) {
   .App-header {
-    flex-direction: column;
+    justify-content: space-between;
+    flex-wrap: wrap;
+
+    .App-header__nav--hidden {
+      display: none;
+    }
+
+    .App-header__hamburger {
+      display: block;
+      text-align: right;
+      user-select: none;
+    }
+
+    .App-header__logo {
+      flex: 0 0 75%;
+    }
+
+    svg {
```

Comment:
Can we please target the menu icon with a class? Just so we avoid side effects on the styles

Reply:
nit: We could also remove hover effect and cursor pointer on the links. Should not be required on mobile

Reply:
not exactly sure how to handle this. media query?

> nit: We could also remove hover effect and cursor pointer on the links. Should not be required on mobile

Reply:
Yes, in the same breakpoint query, inside the nav element, afte [this line](https://github.com/code4romania/taskforce-fe-components/pull/102/files#diff-748c6711719a7c21d1a4684ae78c3987R74) just have something below.
```css
nav {
    a:hover { 
        cursor: auto;
        background: none;
    }
}
```